<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>空载电流测试台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,8">
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_title">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>空载电流测试台</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btn_refresh">
      <property name="text">
       <string>刷新串口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_open">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>COM:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>1</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,1,0,1">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>范围设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>电流上限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>电流下限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_upperlimit1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_lowerlimit1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>测试数据</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>最大电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>最小电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_maxcurrent1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_mincurrent1">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>检测结果</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_12" rowstretch="1,1">
            <item row="1" column="0">
             <widget class="QTextEdit" name="edit_result1"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_icon1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>2</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,1,0,1">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>范围设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>电流下限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>电流上限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_upperlimit2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_lowerlimit2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>测试数据</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9" columnstretch="1,2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>最小电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>最大电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_maxcurrent2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_mincurrent2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>检测结果</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_icon2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="edit_result2"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>3</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,1,0,1">
         <item>
          <widget class="QGroupBox" name="groupBox_11">
           <property name="title">
            <string>范围设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>电流下限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>电流上限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_upperlimit3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_lowerlimit3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="title">
            <string>测试数据</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10" columnstretch="1,2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>最小电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>最大电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_maxcurrent3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_mincurrent3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="title">
            <string>检测结果</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_icon3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="edit_result3"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>4</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,0,1,0,1">
         <item>
          <widget class="QGroupBox" name="groupBox_14">
           <property name="title">
            <string>范围设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>电流上限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>电流下限(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_upperlimit4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_lowerlimit4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_15">
           <property name="title">
            <string>测试数据</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11" columnstretch="1,2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>最大电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>最小电流(A)：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="edit_maxcurrent4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="edit_mincurrent4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_16">
           <property name="title">
            <string>检测结果</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
            <item>
             <widget class="QLabel" name="label_icon4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="edit_result4"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>btn_open</tabstop>
  <tabstop>btn_refresh</tabstop>
  <tabstop>edit_upperlimit1</tabstop>
  <tabstop>edit_lowerlimit1</tabstop>
  <tabstop>edit_upperlimit2</tabstop>
  <tabstop>edit_lowerlimit2</tabstop>
  <tabstop>edit_upperlimit3</tabstop>
  <tabstop>edit_lowerlimit3</tabstop>
  <tabstop>edit_upperlimit4</tabstop>
  <tabstop>edit_lowerlimit4</tabstop>
  <tabstop>edit_maxcurrent1</tabstop>
  <tabstop>edit_mincurrent1</tabstop>
  <tabstop>edit_maxcurrent2</tabstop>
  <tabstop>edit_mincurrent2</tabstop>
  <tabstop>edit_maxcurrent3</tabstop>
  <tabstop>edit_mincurrent3</tabstop>
  <tabstop>edit_maxcurrent4</tabstop>
  <tabstop>edit_mincurrent4</tabstop>
  <tabstop>edit_result1</tabstop>
  <tabstop>edit_result2</tabstop>
  <tabstop>edit_result3</tabstop>
  <tabstop>edit_result4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
